platformio.ini

```diff
@@ -41,16 +41,14 @@ extra_scripts =
 build_unflags = -std=gnu++11 # The correct flag will be set by the cppflags python script...
 
 ; Light edition by hardware revisions
-[env:LIGHT_EDITION_V3_2]
```

Comment:
Just to be safe: you checked the docs and other documents (like the README) for any mentions?